### PR TITLE
[M1733] implement prod cdk

### PR DESCRIPTION
**Pull request by aspcanada**

### CODE REVIEW

**Comment by aspcanada** [comment]
[Implement PROD cdk deployment](https://trello.com/c/EWUO27K4/1733-implement-prod-cdk-deployment)

File: iac/settings/prod.py [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+    api=DjangoSettings(
+        container_cpu=1024,
+        container_memory=2048,
+        container_count=1,
+        default_domain_api="api.datamermaid.org",
+        default_domain_collect="collect.datamermaid.org",
+        mermaid_api_audience="https://api.datamermaid.org",
+        
+        # Secrets
+        # dev_emails_name="dev/mermaid-api/dev-emails-mUnSDl"
```
Comment (gridcell) [inline iac/settings/prod.py:18]: THis need to be here?

Comment (aspcanada) [reply iac/settings/prod.py:18]: nope!

Comment (aspcanada) [reply iac/settings/prod.py:18]: I'll remove it on the next one

**Pull request merged**